app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+    OutdatedVaccineError, NotWearingMaskError

```

Comment:
```suggestion
from app.errors import (
    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError
)
```
Better to use brackets instead of \
Also you can do:
```
from app.errors import (
    NotVaccinatedError, 
    OutdatedVaccineError, 
    NotWearingMaskError
)
```